internal/sbi/consumer/nrf_service.go

```diff
@@ -2,22 +2,20 @@ package consumer
 
 import (
 	"context"
-	"fmt"
-	"net/http"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/antihax/optional"
 	"github.com/pkg/errors"
 
 	ausf_context "github.com/free5gc/ausf/internal/context"
 	"github.com/free5gc/ausf/internal/logger"
 	"github.com/free5gc/openapi"
-	"github.com/free5gc/openapi/Nnrf_NFDiscovery"
-	"github.com/free5gc/openapi/Nnrf_NFManagement"
 	"github.com/free5gc/openapi/models"
+	Nnrf_NFDiscovery "github.com/free5gc/openapi/nrf/NFDiscovery"
+	"github.com/free5gc/openapi/nrf/NFManagement"
```

Comment:
remove redundancy import

Reply:
I tried to modify it, but it would pop up again after I pressed ctrl+s to save. Lol

Reply:
Please choose one (`NFManagement` or `Nnrf_NFManagement`) to represent "github.com/free5gc/openapi/nrf/NFManagement"